omod-2.0/pom.xml

```diff
@@ -142,6 +142,12 @@
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>legacyui-omod</artifactId>
+			<version>1.3.3</version>
```

Comment:
Add scope as provided.

Reply:
Here: https://github.com/openmrs/openmrs-module-webservices.rest/pull/484/commits
Failing resource tests in omod-2.1+ as they can't see `AdministrationSectionExt` class.

Reply:
Brought by just adding scope provided?

Reply:
It seems so, before adding the provided scope there was only PersonResourceTest failing with a different reason

Reply:
@dkayiwa So I found a way to make it work. I've added `legacyui-omod` as a test dependency to all 2.0+ modules. I also had to bump it to version 1.8.3. Now it passes all the tests and can be deployed, the endpoint runs as well.

The only concern I have is why the `PersonResource2_2Test` failed on display property? It seems that adding `legacyui-omod` modified Person object for test purposes.

https://github.com/openmrs/openmrs-module-webservices.rest/pull/484/commits/03f6a26c82cb988f36f68b7e392f4f8faeb1d613